org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -2070,3 +2075,20 @@ SsoOpenchallengesProdAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref OpenchallengesProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoSynapseLlmDevAdmin:
```

Comment:
this is the `personal AWS accounts` section.  please move this about personal accounts.

Reply:
Moved with the other Synapse* access